plugins/offers_offer.c

```diff
@@ -467,7 +467,7 @@ struct command_result *json_invoicerequest(struct command *cmd,
 	json_add_bool(req->js, "exposeid", true);
 	json_add_bool(req->js, "single_use", *single_use);
 	if (label)
-		json_add_string(req->js, "label", label);
+		json_add_string(req->js, "recurrence_label", label);
```

Comment:
Yes, though we could simply rename the createinvoice interface here, since in this case it's not being used for recurrence.

Not now though.